client/my-sites/checkout/checkout-thank-you/business-plan-details.jsx

```diff
@@ -80,6 +81,18 @@ const BusinessPlanDetails = ( {
 				/>
 			) }
 
+			{ ! selectedFeature && (
+				<PurchaseDetail
+					icon={ <img alt="" src={ themeImage } /> }
+					title={ i18n.translate( 'Try a New Theme' ) }
+					description={ i18n.translate(
+						"You've now got access to every premium theme, at no extra cost. Give one a try!"
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'You've now got access to every premium theme, at no extra cost - that's hundreds of new options. Give one a try!' )` *ES Score: 7*